app/components/disciplines.cjsx

```diff
@@ -0,0 +1,15 @@
+module.exports =
```

Comment:
Just noticed this is in `components` but isn't a component. Maybe move to `lib`.